.github/workflows/test.yaml

```diff
@@ -14,6 +14,15 @@ on:
         type: string
 
 jobs:
+  conda-cpp-checks:
```

Comment:
Let's be consistent in whether we define `conda-cpp-checks` above or below `conda-cpp-tests` in `pr.yaml` and `test.yaml`.

Reply:
Done